src/plugins/intel_npu/src/compiler_adapter/include/zero_adapter.hpp

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ze_api.h>
+#include <ze_graph_ext.h>
+
+#include <type_traits>
+#include <utility>
+
+#include "intel_npu/icompiler.hpp"
```

Comment:
This can be removed now. That was the objective.

Reply:
done